samples/Samples.Server30/Startup.cs

```diff
@@ -0,0 +1,90 @@
+using GraphQL.Samples.Schemas.Chat;
+using GraphQL.Server;
+using GraphQL.Server.Ui.GraphiQL;
+using GraphQL.Server.Ui.Playground;
+using GraphQL.Server.Ui.Voyager;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+
+namespace GraphQL.Samples.Server
```

Comment:
I would prefer not to copy this file but to manage one. This will be a good example of source level backward compatibility.

Reply:
Do you want me to Include Existing as Link for Program.cs and Startup.cs? 

There's quite a few changes as some core types change. I can see the need to not manage two files, but I can also see that we don't wanna scare people off with compiler conditionals. There's a case to be made for both :P Let me know.

Reply:
> Do you want me to Include Existing as Link for Program.cs and Startup.cs?

Yes. 

> There's quite a few changes as some core types change. I can see the need to not manage two files, but I can also see that we don't wanna scare people off with compiler conditionals. 

Agree. If the code is very different for the two platforms, then using the preprocessor makes it an unreadable stuff. But if you only need a couple of such switches, then using one file makes sense - the reader focuses on the switch point. In the case of different examples, he does not even bother to see how they differ. I am for the thoughtful reader to understand what the difference is right away, and not be forced to conduct a manual comparison.

Reply:
Done!